department-of-veterans-affairs/abd-vro
# Update & confirm state of claim/contention fields off-ramp/order-exam/ready for decisioni

## What was the problem?


The following tickets describe it well. 

https://github.com/department-of-veterans-affairs/abd-vro/issues/1286
https://github.com/department-of-veterans-affairs/abd-vro/issues/1287
https://github.com/department-of-veterans-affairs/abd-vro/issues/1288

Associated tickets or Slack threads:

- [MCP-000](https://amida.atlassian.net/browse/MCP-000)

## How does this fix it?[^1]


Implements functionality as described in the above tickets. Error condition (due to BIP API errors or fields changing during VRO processing is not addressed here. They will be addressed in other PRs.

## How to test this PR
- Make sure end-to-end tests provide enough coverage for all the cases and they pass
- Step 2[^secrel]


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
